File: android/app/src/main/java/com/oandmtech/schoolvillage/PdfViewerActivity.java
package com.oandmtech.schoolvillage;

import android.app.Activity;
import android.os.Bundle;

import com.github.barteksc.pdfviewer.PDFView;
import com.github.barteksc.pdfviewer.util.FitPolicy;

import java.io.File;

public class PdfViewerActivity extends Activity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_pdf_viewer);
        PDFView pdfView = findViewById(R.id.pdfView);

        pdfView.fromFile(new File(getIntent().getStringExtra("URL")))
                .pageFitPolicy(FitPolicy.WIDTH)
                .spacing(12)
                .load();
    }

}

File: android/app/src/main/kotlin/com/oandmtech/schoolvillage/PdfViewerActivity.kt
package com.oandmtech.schoolvillage

import com.github.barteksc.pdfviewer.PDFView
import com.github.barteksc.pdfviewer.util.FitPolicy
import android.app.Activity
import android.os.Bundle

class PdfViewerActivity : Activity() {

    protected override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_pdf_viewer)
        val pdfView: PDFView = findViewById(R.id.pdfView)
        pdfView.fromFile(java.io.File(getIntent().getStringExtra("URL")))
                .pageFitPolicy(FitPolicy.WIDTH)
                .spacing(12)
                .load()
    }

}